src/ccmain/tesseractclass.cpp

```diff
@@ -63,8 +63,11 @@ Tesseract::Tesseract()
                   "Break input into lines and remap boxes if present", this->params())
     , BOOL_MEMBER(tessedit_dump_pageseg_images, false,
                   "Dump intermediate images made during page segmentation", this->params())
-    , BOOL_MEMBER(tessedit_do_invert, true, "Try inverting the image in `LSTMRecognizeWord`",
+    , BOOL_MEMBER(tessedit_do_invert, true, "Try inverted line image if necessary",
```

Comment:
What about adding to the parameter description: 

`This parameter is deprecated and will be removed in version 6.0. Use the 'invert_threshold' parameter instead.`

Also add:

`// DEPRECATED`

or
```
// TODO (Stefan)
// Remove 'do_invert' in 6.0.
```
comment above.


This parameter deprecation should be mentioned in the release notes.

Reply:
This PR was merged. Can you please push the suggested changes as a new commit?

Reply:
Done, see commit b8b6c158a7f356971d7ffc76b6d688606e05690d.

Reply:
Thanks.